pallets/parachain-staking/src/tests.rs

```diff
@@ -4143,25 +4046,42 @@ fn parachain_bond_inflation_reserve_matches_config() {
 					total_balance: 140,
 				},
 				Event::Rewarded {
+					/*Explanation of how reward is computed:
+						Total staked = 50
+						Total reward to be paid = 50
+						Collator stake = 20
+
+						collator gets 20% commission = 10 (20% of [total reward])
+						Reward left to distribute = 40 (50 - 10)
+						collator gets 40% ([collator stake] * 100 / [total staked]) of the [reward left] = 16 (40 * 40 / 100)
+						Total 10 + 16 = 26 [commision + collator share]
+					*/
 					account: 1,
-					rewards: 20,
+					rewards: 26,
```

Comment:
Values have changed because we don't "keep" a portion of the reward for parachain bonds anymore.

Reply:
Could you please make a note that this is subject to change, since these values agree to the current staking logic (Moonbeam's) but ours will be different?